app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find_by(params[:id])
+  end
+end
```

Comment:
- Kindly ensure you have `posts_controller` with `#index` and `#show` actions in your controllers. 👍🏽

Reply:
Okay, I see. Thanks